prediction-applier-extension/pom.xml

```diff
@@ -38,11 +42,32 @@
             <groupId>org.alfresco</groupId>
             <artifactId>alfresco-hxinsight-connector-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.alfresco</groupId>
+            <artifactId>alfresco-hxinsight-connector-common-authentication</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.camel.springboot</groupId>
+                    <artifactId>camel-http-starter</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency> <!-- Camel version in extension has to be the same as in ACS -->
```

Comment:
This comment should probably go in the `properties` section now. Can this work against multiple different versions of ACS??

Reply:
It passed E2E tests against 7.3.2 and 7.4.2, seems it works fine either with older versions. I will adjust the comment

Reply:
I've verified it and found out that it's possible to run the Repo extension with older versions of ACS (ex. 7.3.2) but all these dependiences:
```
<include>jakarta.servlet:jakarta.servlet-api</include>
<include>org.apache.camel:camel-http-base</include>
<include>org.apache.camel:camel-http-common</include>
<include>org.apache.camel:camel-http</include>
<include>org.apache.httpcomponents.core5:httpcore5</include>
<include>org.apache.httpcomponents.core5:httpcore5-h2</include>
<include>org.apache.httpcomponents.client5:httpclient5</include>
```
had to be included into jar